src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  todos: Todo[],
+  setFilterStatus: (filter: Filter) => void,
+  isCompleted: boolean,
+  filterStatus: Filter,
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  todos,
+  setFilterStatus,
+  isCompleted,
+  filterStatus,
+}) => {
+  const activeTodos = [...todos].filter(todo => !todo.completed);
```

Comment:
The filter method returns a copy of the array, so you don't need to do it manually
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter
```suggestion
  const activeTodos = todos.filter(todo => !todo.completed);
```

Reply:
Not fixed 😢

Reply:
> The filter method returns a copy of the array, so you don't need to do it manually https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Here as well